clientapp/src/components/TracerForm.jsx

```diff
@@ -116,12 +121,16 @@ export class TracerForm extends React.Component {
     }
 
     onStartDateChange(ev) {
-        const newStartDate = ev.target.value;
+        var startDate = document.getElementById("start-date").value;
```

Comment:
this acts weird if you pick a time before a date. 
![image](https://user-images.githubusercontent.com/16966368/88840832-ee7f3a80-d191-11ea-9464-3cdc39291f1b.png)